Hybrid Search Revamped article (#1041)
qdrant/landing_page: qdrant-landing/content/articles/hybrid-search.md

date: 2023-02-15T10:48:00.000Z
---

<aside role="status">
This article was published in 2023, more than a year before the Qdrant 1.10 release introducing the new 

> <a href="/documentation/concepts/search/#query-api">Query API</a>. Although the presented concepts are still valid, 

Reviewer: ```suggestion
<a href="/documentation/concepts/search/#query-api">Query API</a>. Although the concepts in this article are still valid, 
```

> Qdrant 1.10 makes it easier to build a hybrid search system, without any additional tools in the stack. 

Reviewer: ```suggestion
you should use  Qdrant 1.10 to build a hybrid search system. It is easier and requires no additional tools from your stack. 
```
